.shippable.yml

```diff
@@ -40,16 +40,7 @@ build:
             echo "Building a Pull Request";
             echo "- Building Documentation";
             echo "Commit range:" ${COMMIT_RANGE}
-            make htmldocs > doc.log 2>&1;
-            ./scripts/filter-known-issues.py --config-dir .known-issues/doc/ doc.log > doc.warnings;
-
-            if [ "$?" != 0 ]; then
-              echo " ==> Error running filter script"
-              exit 1
-            fi;
-            if [ -s doc.warnings ]; then
-              echo " => New documentation warnings/errors";
-            fi;
+            make htmldocs
```

Comment:
the script check-compliance below expects to find the output of filter-known-issues in 

DOCS_WARNING_FILE = "doc.warnings"

So this will not work and doc issues wont be reported by shippable.

Reply:
Oh.  Didn't know that.  I just had the script return a status code on error.